packages/js/src/plugins/auctionHouseModule/updateAuctionHouse.ts

```diff
@@ -36,6 +41,8 @@ export type UpdateAuctionHouseInput = {
   newAuthority?: PublicKey;
   feeWithdrawalDestination?: PublicKey;
   treasuryWithdrawalDestinationOwner?: PublicKey;
+  auctioneerAuthority?: Signer;
```

Comment:
IIRC, this doesn't need to be a signer.

```suggestion
  auctioneerAuthority?: PublicKey;
```